### PR TITLE
fix: npm security issues by upgrading electron packages and nodejs version

### PR DESCRIPTION
**Pull request by martiliones**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import store from './store/index.js'
 import { i18n } from './i18n'
 import VueFormatters from './lib/formatters'
 import packageJSON from '../package.json'
-import { vuetify } from '@/plugins/vuetify'
+import { vuetify } from '@/plugins/vuetify.ts'
```
Comment (bludnic) [inline src/main.ts:9]: why you added import extension?

Comment (adamant-al) [reply src/main.ts:9]: It seems `[fix: vite cannot resolve typescript files](https://github.com/Adamant-im/adamant-im/pull/499/commits/ca78437c76ef017b1f1a8afca4d639860232c0de)`

**Comment by adamant-al** [comment]
- Consider to update this one in the PR also: `vite-plugin-electron              ^0.11.1  →   ^0.12.0`
- Unresolved conflict: `src/App.vue`
- @bludnic Make sure electron apps for macOS are building successfully with the changes, and the app runs. For me, I see `Приложение «ADAMANT Messenger» неожиданно завершило работу.` when I run `./release-electron/mac-arm64/ADAMANT Messenger.app`

**Comment by adamant-al** [comment]
Upd: Same `./release-electron/mac-arm64/ADAMANT Messenger.app` build from the `master` branch crashes on my computer also. Maybe, the reason in my system.
Try it.

**Review by joebrock666 — COMMENTED** [review]
fix the conflict otherwise all looks good

**Comment by bludnic** [comment]
@martiliones check Trello cards

File: src/main.ts (hunk 1 above)
Comment (bludnic) [reply src/main.ts]: Solved by resolving `.ts` extension in vite.config.ts https://github.com/Adamant-im/adamant-im/pull/499/commits/2fbe4663b064443c205cf5b49ed876b1abde76c6

**Review by adamant-al — APPROVED** [review]

**Review by joebrock666 — APPROVED** [review]

**Pull request merged**